samples/Azure.Management.Storage/Generated/BlobService.cs

```diff
@@ -88,6 +88,32 @@ public virtual BlobContainerCollection GetBlobContainers()
         {
             return new BlobContainerCollection(Client, Id);
         }
+        /// <summary>
+        /// Gets properties of a specified container. 
+        /// Request Path: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName}
+        /// Operation Id: BlobContainers_Get
+        /// </summary>
+        /// <param name="containerName"> The name of the blob container within the specified storage account. Blob container names must be between 3 and 63 characters in length and use numbers, lower-case letters and dash (-) only. Every dash (-) character must be immediately preceded and followed by a letter or number. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentException"> <paramref name="containerName"/> is an empty string, and was expected to be non-empty. </exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="containerName"/> is null. </exception>
+        public async virtual Task<Response<BlobContainer>> GetBlobContainerAsync(string containerName, CancellationToken cancellationToken = default)
```

Comment:
This should go on the parent resource to live along side the GetBlobContainers()